GoogleDataTransport/GDTCCTLibrary/GDTCCTUploadOperation.m

```diff
@@ -78,14 +65,24 @@ @interface GDTCCTUploadOperation () <NSURLSessionDelegate>
 
 @property(nonatomic, readonly) GDTCORTarget target;
 @property(nonatomic, readonly) GDTCORUploadConditions conditions;
+@property(nonatomic, readonly) NSURL *uploadURL;
+@property(nonatomic, nullable, readonly) NSString *APIKey;
+@property(nonatomic, readonly) id<GDTCCTUploadMetadataProvider> metadataProvider;
 
 @property(nonatomic, readwrite, getter=isExecuting) BOOL executing;
 @property(nonatomic, readwrite, getter=isFinished) BOOL finished;
+
+@property(nonatomic, copy) dispatch_block_t block;
```

Comment:
maybe a more descriptive name?

Reply:
Ah, I'll remove it. I accidentally committed it instead of reverting.  It was a bad idea to demonstrate some Objective-C specifics in a real project 😄